common/sai.py

```diff
@@ -1,180 +1,231 @@
-from enum import Enum
-import redis
-import time
 import json
+import logging
 import os
 import pytest
 
+from saichallenger.common.sai_abstractions import AbstractEntity
+from saichallenger.common.sai_client.sai_client import SaiClient
+from saichallenger.common.sai_data import SaiObjType
+
+
+class Sai(AbstractEntity):
```

Comment:
The `AbstractEntity` contains `init_connections()` which is not related to SAI interface. So, `Sai` should not be inherited from `AbstractEntity`.